packages/@sanity/base/src/time/useTimeAgo.ts

```diff
@@ -48,33 +48,34 @@ export function useTimeAgo(time: Date | string, {minimal}: TimeAgoOpts = {}): st
 // eslint-disable-next-line complexity
```

Comment:
I think the complexity here is now allowed by our updated eslint config, could you remove this while you're in here? :)